src/App.jsx

```diff
@@ -1,60 +1,268 @@
-import "./App.css"
+import { useState } from "react";
+import "./App.css";
 
 function App() {
+	const [firstNumber, setFirstNumber] = useState("0");
+	const [secondNumber, setSecondNumber] = useState("0");
+	const [operator, setOperator] = useState("+");
+	const [total, setTotal] = useState("0");
+	const [storedNumber, setStoredNumber] = useState("0");
 
-  return (
-    <div className="calculator">
-      <div className="panel">
-        <p>0</p>
-        <div className="numbers">
-          <button>1</button>
-          <button>2</button>
-          <button>3</button>
-          <button>4</button>
-          <button>5</button>
-          <button>6</button>
-          <button>7</button>
-          <button>8</button>
-          <button>9</button>
-          <button>0</button>
-          <button>Clear</button>
-        </div>
-      </div>
-
-      <div className="panel">
-        <p>+</p>
-        <div className="numbers">
-          <button>+</button>
-          <button>-</button>
-          <button>*</button>
-          <button>÷</button>
-        </div>
-      </div>
-
-      <div className="panel">
-        <p>0</p>
-        <div className="numbers">
-          <button>1</button>
-          <button>2</button>
-          <button>3</button>
-          <button>4</button>
-          <button>5</button>
-          <button>6</button>
-          <button>7</button>
-          <button>8</button>
-          <button>9</button>
-          <button>0</button>
-          <button>Clear</button>
-        </div>
-      </div>
-      <div className="panel answer">
-        <p>0</p>
-        <div>
-          <button>=</button>
-        </div>
-      </div>
-    </div>
-  )
+	const calculate = (firstNumber, secondNumber, operator) => {
+		firstNumber = parseFloat(firstNumber);
+		secondNumber = parseFloat(secondNumber);
+
+		switch (operator) {
+			case "+":
+				return firstNumber + secondNumber;
+			case "-":
+				return firstNumber - secondNumber;
+			case "*":
+				return firstNumber * secondNumber;
+			case "÷":
+				return firstNumber / secondNumber;
+			default:
+				throw console.error("Not a valid operator");
+		}
+	};
+
+	const updatePanelNumber = (currentNumber, newNumber) => {
```

Comment:
this function is misleading, it's not updating the panel number, it's instead computing the updated panel number

rename to `computeUpdatedNumber`